shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +76,77 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+// Creates and initializes a UIVisualEffectView with a UIBlurEffect. Extracts and returns its
+// gaussianFilter. Logs errors and returns if Apple's API has changed and the filter can't be
+// extracted.
+- (NSObject*)extractGaussianFilter {
```

Comment:
nit: maybe make this static? This doesn't seem to make use of any instance variables.

Also when this method returns `nil` we probably don't have to try a second time since it's almost guaranteed to fail.